packages/vscode/src/plugins/nextjs/index.ts

```diff
@@ -0,0 +1,95 @@
+import {
+  commands,
+  TextDocument,
+  TextDocumentSaveReason,
+  TextDocumentWillSaveEvent,
+  window,
+  workspace,
+} from 'vscode'
+import { packageJsonIncludes } from '../../helpers/workspace'
+import { PrismaVSCodePlugin } from '../types'
+import { NextTypes } from './nextTypes'
+
+const nextTypes = new NextTypes({ save: true })
+const supportedLanguageIds = [
+  'typescriptreact',
+  'javascriptreact',
+  'typescript',
+  'javascript',
+]
+// This is used to get around other plugins that modify the document in onWillSave
+let shouldUpdate = false
+
+const plugin: PrismaVSCodePlugin = {
+  name: 'nextjs',
+  enabled: async () => {
+    // TODO Add Workspace Support currently only checks the workspace root for deps
```

Comment:
Did you create an issue for this todo?

Reply:
https://github.com/prisma/language-tools/issues/556